Makefile

```diff
@@ -137,20 +108,16 @@ test-docker:
 	@echo ">> testing docker image"
 	./test_image.sh "$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)" 9100
 
-$(GOPATH)/bin/promtool promtool:
+$(FIRST_GOPATH)/bin/promtool promtool:
 	@GOOS= GOARCH= $(GO) get -u github.com/prometheus/prometheus/cmd/promtool
 
-$(GOPATH)/bin/promu promu:
+$(FIRST_GOPATH)/bin/promu promu:
 	@GOOS= GOARCH= $(GO) get -u github.com/prometheus/promu
 
-$(GOPATH)/bin/staticcheck:
-	@GOOS= GOARCH= $(GO) get -u honnef.co/go/tools/cmd/staticcheck
-
-
 .PHONY: all style format build test test-e2e vet tarball docker promtool promu staticcheck checkmetrics
 
 # Declaring the binaries at their default locations as PHONY targets is a hack
 # to ensure the latest version is downloaded on every make execution.
 # If this is not desired, copy/symlink these binaries to a different path and
 # set the respective environment variables.
-.PHONY: $(GOPATH)/bin/promtool $(GOPATH)/bin/promu $(GOPATH)/bin/staticcheck
+.PHONY: $(FIRST_GOPATH)/bin/promtool $(FIRST_GOPATH)/bin/promu $(FIRST_GOPATH)/bin/staticcheck
```

Comment:
With the updated Makefile.common, this can be simplified to:

```
.PHONY: $(FIRST_GOPATH)/bin/promtool
```

Reply:
thank you, fixed